cassava_trait.obo

```diff
@@ -8770,6 +8769,876 @@ relationship: variable_of CO_334:0013012 ! attieke-like aroma method
 relationship: variable_of CO_334:0103029 ! Attieke aroma scale 0-10
 created_by: Ebah Catherine
 
+[Term]
+id: CO_334:0003144
+name: Canopy formation Tricot block 3
+namespace: cassava_trait
+synonym: "CanForm_CompRank_Bl3" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003203 ! Canopy formation
+relationship: variable_of CO_334:0013054 ! Comparative ranking by users method
+relationship: variable_of CO_334:0103077 ! Block:3
+
+[Term]
+id: CO_334:0003145
+name: Discoloration Tricot block 3
+namespace: cassava_trait
+synonym: "Discol_CompRank_Bl3" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003204 ! Discoloration
+relationship: variable_of CO_334:0013054 ! Comparative ranking by users method
+relationship: variable_of CO_334:0103077 ! Block:3
```

Comment:
all the variables with 'block 3' scale should have the actual scale used for measuring each trait. Whether it's a quantitative unit or a nominal scale, etc. 

I also suggest to completely omit 'block 3' from the variables to make them reusable by others.